include/arch/arm/aarch32/cortex_m/scripts/linker.ld

```diff
@@ -33,25 +34,6 @@
 #define _DATA_IN_ROM
 #endif
 
-#if !defined(CONFIG_XIP) && (CONFIG_FLASH_SIZE == 0)
-#define ROM_ADDR RAM_ADDR
-#else
-#define ROM_ADDR (CONFIG_FLASH_BASE_ADDRESS + CONFIG_FLASH_LOAD_OFFSET)
-#endif
-
-#ifdef CONFIG_HAS_TI_CCFG
-#define CCFG_SIZE 88
-#define ROM_SIZE (CONFIG_FLASH_SIZE*1K - CONFIG_FLASH_LOAD_OFFSET - \
-	CCFG_SIZE)
-#define CCFG_ADDR (ROM_ADDR + ROM_SIZE)
```

Comment:
`CCFG_ADDR` is still being used in the code below:
```
#ifdef CONFIG_HAS_TI_CCFG
    FLASH_CCFG            (rwx): ORIGIN = CCFG_ADDR, LENGTH = CCFG_SIZE
#endif
```

causing:
```
invalid origin for memory region FLASH_CCFG
collect2: error: ld returned 1 exit status
ninja: build stopped: subcommand failed.

```

Reply:
Seems to be corrected in: https://github.com/zephyrproject-rtos/zephyr/pull/34185
So assume this is now pending a rebase after #34185 is merged.